single-rai-job/src/run_rai.py

```diff
@@ -143,6 +148,7 @@ def add_properties_to_gather_run(
     run_properties = {
         PropertyKeyValues.RAI_INSIGHTS_TYPE_KEY: PropertyKeyValues.RAI_INSIGHTS_TYPE_GATHER,
         PropertyKeyValues.RAI_INSIGHTS_RESPONSIBLEAI_VERSION_KEY: responsibleai_version,
+        PropertyKeyValues.RAI_INSIGHTS_DATA_TYPE_KEY: FORECASTING_DATA_TYPE,
```

Comment:
shouldn't this only be conditionally added for forecasting scenarios? it seems like it's added for all tabular scenarios here.

Reply:
good point, just fixed